ecosystem/typescript/sdk/src/aptos_client.ts

```diff
@@ -40,6 +40,14 @@ export class AptosClient {
     } else {
       conf.BASE = fixNodeUrl(nodeUrl);
     }
+
+    // Do not carry cookies when `WITH_CREDENTIALS` is explicitly set to `false`. By default, cookies will be sent
```

Comment:
conf.WITH_CREDENTIALS = !!conf?.WITH_CREDENTIALS;

Reply:
yeah, but want to be explicit here.